tests/e2e/autoscale/01-check-simplest-collector-hpa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: go run ../../../hack/wait-until-hpa-ready/main.go --hpa simplest-collector
```

Comment:
I would maybe prefer to host the "script" in this directory along the tests since it's used only here.